app/src/main/java/org/simple/clinic/summary/teleconsultation/api/TeleconsultationsResponse.kt

```diff
@@ -5,6 +5,12 @@ import com.squareup.moshi.JsonClass
 
 @JsonClass(generateAdapter = true)
 data class TeleconsultationsResponse(
+    @Deprecated(
+        message = "Use teleconsultationPhoneNumbers to get list of all MO phone numbers",
+        replaceWith = ReplaceWith("teleconsultationPhoneNumbers", "org.simple.clinic.summary.teleconsultation.api")
+    )
     @Json(name = "teleconsultation_phone_number")
-    val teleconsultationPhoneNumber: String?
+    val teleconsultationPhoneNumber: String?,
+    @Json(name = "teleconsultation_phone_numbers")
+    val teleconsultationPhoneNumbers: List<TeleconsultPhoneNumber>?
```

Comment:
Is the API spec shared somewhere? I could neither see it on the PRD nor on the API docs.

Reply:
Yeah, it's been shared on slack. 
https://simpledotorg.slack.com/archives/C012M349Q4X/p1591261415030900